### ISSUE TITLE
Eval of pretrained models

### ISSUE THREAD

**Issue by prshnthrv** [karpathy/nanoGPT#245]
If I run
$ python train.py config/eval_gpt2.py
but change the dropout value in https://github.com/karpathy/nanoGPT/blob/master/train.py#L55 to 0.1 or 0.2, the val_loss is different.
For example,
dropout = 0, I get a val_loss of 3.09, which is close to what you get.
If I set dropout = 0.1 and run eval again, I get a val_loss of 3.49
With dropout = 0.2, val_loss is 4.23.
Any idea why? I stepped through and indeed the model is in eval() mode and dropout should not have any impact during evaluation, correct? What am I missing?

**Comment by YassineYousfi** [comment]
I can't reproduce this.
With eval_iters = 100 I get the same loss with dropout = 0 and 0.2
```step 0: train loss 3.1253, val loss 3.1237```
